homeworks/nazar_kostiv/HW04/tests.py

```diff
@@ -0,0 +1,30 @@
+import sys
+import argparse
+from pyats.aetest import Testcase, test, main, loop, setup
+
+mapping = {
+    'a': (1, 3, 4, 5, 6, 7, 8),
+    'b': (0, 2, 3, 4),
+    'c': (7, 9, 0, 6, 5, 4, 3, 1)
+}
+
+parser = argparse.ArgumentParser(description="Parsing letter")
```

Comment:
@ProffesorRabbit please move it to main section.

Reply:
@extsoft fixed